src/rules/no-unused-modules.js

```diff
@@ -0,0 +1,720 @@
+/**
+ * @fileOverview Ensures that modules contain exports and/or all 
+ * modules are consumed within other modules.
+ * @author René Fermann
+ */
+
+import Exports from '../ExportMap'
+import { makeOptionsSchema } from 'eslint-module-utils/moduleVisitor'
+import resolve from 'eslint-module-utils/resolve'
+import docsUrl from '../docsUrl'
+
+// eslint/lib/util/glob-util has been moved to eslint/lib/util/glob-utils with version 5.3
```

Comment:
Can you file an upstream issue? This is a breaking change in a minor version.

Reply:
```listFilesToProcess``` is not part of the official ESLint Node.js API. Hence it is not a breaking change (at least thats how I interpret semantic versioning, please correct me, if I'm wrong in this point).

Additionally, the version of ESLint, in which this change gets introduced, doesn't matter. I still need to make sure ```listFilesToProcess``` gets correctly imported.

Reply:
It's debatable - to me, if it's reachable, it's part of the public API.

I'd still like to see an upstream issue filed; they should provide the `eslint/lib/util/glob-util` path until v6.

Reply:
What difference does it make for us if they revert the change for now and introduce it again with v6? This rule still needs to handle both cases in order to stay backwards compatible.

Don't get me wrong, I don't refuse to argue for something, which has a benefit for this library. But I struggle to see a benefit in arguing with the eslint team in this particular point.

Reply:
@ljharb: what do you think about my last comment?

Reply:
@rfermann That's a good point - but I think that if they release a bugfix in eslint 5, then we don't actually have to support this rule on an un-patched eslint 5.

Reply:
Thanks, filing the issue and linking it here would be great.